File: prak_lat1 cpp/Processor.cpp
#include <iostream>
#include <string.h>

using namespace std;

class Processor{
	private:
		//Attributes
		string name;
		int price;

	public:
		// Constructors
		Processor(){}
		Processor(string name, int price){
			this->name = name;
			this->price = price;
		}

		// Get & Set methods
		void setName(char name[]){
			this->name = name;
		}

		void setPrice(int price){
			this->price = price;
		}

		string getName(){
			return this->name;
		}

		int getPrice(){
			return this->price;
		}

		~Processor(){}
};

File: prak_lat1 java/Processor.java
public class Processor{

	// Attributes
	private String name;
	private int price;

	// Constructor
	public Processor(String name, int price){
		this.name = name;
		this.price = price;
	}

	// Set & Get Methods
	public void setName(String name){
		this.name = name;
	}

	public void setPrice(int price){
		this.price = price;
	}

	public String getName(){
		return this.name;
	}

	public int getPrice(){
		return this.price;
	}
}